diesel_tests/Cargo.toml

```diff
@@ -12,7 +12,7 @@ dotenv = ">=0.8, <0.11"
 [dependencies]
 assert_matches = "1.0.1"
 chrono = { version = "0.4" }
-diesel = { path = "../diesel", default-features = false, features = ["quickcheck", "chrono", "uuid", "serde_json", "network-address", "numeric"] }
+diesel = { path = "../diesel", default-features = false, features = ["quickcheck", "chrono", "uuid", "serde_json", "network-address", "numeric", "with-deprecated"] }
```

Comment:
Is this the first deprecated stuff that is actually tested ? Weird

Reply:
Everything else we've deprecated is tested through doctests, not integration tests.